docs/features/software-catalog/descriptor-format.md

```diff
@@ -35,8 +35,32 @@ we recommend that you name them `catalog-info.yaml`.
 
 ## Overall Shape Of An Entity
 
-The following is an example of the shape of an entity as returned from the
-software catalog API.
+The following is an example of a descriptor file for a Component entity:
+
+```yaml
```

Comment:
Since this page is all about descriptor formats, it seemed better to include the YAML as grounding _before_ the JSON API representation. Just swapped the order here.